homeassistant/components/zha/core/registries.py

```diff
@@ -200,10 +217,17 @@ def get_entity(
         return default
 
     def strict_match(
-        self, component: str, rule: MatchRule
+        self,
+        component: str,
+        channel_names: Union[Callable, Set[str], str] = set(),
```

Comment:
from what I am reading it seems like these should default to `None` then we should test them for `None` and assign a new set to them if needed in the method itself.

Reply:
Yeah. With every PR I learn something new :D

Reply:
updated.